docker/README.md

```diff
@@ -6,6 +6,7 @@ This Docker image is designed to provide the following
  catalogs
 * An image that can be deployed as a full cluster by mounting in configuration
 * An image to be used as the basis for the Kubernetes Presto operator
+* Distribute an experimental image supporting ARM64 architecture
```

Comment:
Let's remove this line, since it's unrelated to the functionality provided by the image